lib/child_process.js

```diff
@@ -266,7 +262,8 @@ function execFile(file /* , args, options, callback */) {
     uid: options.uid,
     shell: options.shell,
     windowsHide: !!options.windowsHide,
-    windowsVerbatimArguments: !!options.windowsVerbatimArguments
+    windowsVerbatimArguments: !!options.windowsVerbatimArguments,
+    signal: options.signal
```

Comment:
nit: it'd be nice to keep the alphabetical order

```suggestion
    uid: options.uid,
    shell: options.shell,
    signal: options.signal,
    windowsHide: !!options.windowsHide,
    windowsVerbatimArguments: !!options.windowsVerbatimArguments
```

Reply:
Should I also re-order the `uid` and option?

Reply:
Done, please see f642015